client/src/models/verity.js

```diff
@@ -43,13 +43,27 @@ const verity = {
 		/*
 		 * The default value means: image not approved yet, and not rejected yet too
 		 */
-		filter		: new FilterModel({
-			approved		: false,
-			active		: true,
-		}),
-		treeCount: 0,
+		filter: new FilterModel({
+			approved: false,
+			active: true,
+    }),
+    verifiedFilter: new FilterModel({
+      approved: true,
+      active: true,
+    }),
+    rejectedFilter: new FilterModel({
+      approved: false,
+      active: false
```

Comment:
I believe `active` should be set to `true` here, but will check with @deepwinter

Reply:
Looking at the api call, when reject tree is called, active is set to false. 
See here: https://github.com/Greenstand/treetracker-admin/blob/a6f7a931e3d3732f71373afaa2f9c4b980829a06/client/src/api/treeTrackerApi.js#L78
Line 88 below it.

Reply:
Comment, @nmcharlton ?

Reply:
My apologies, you are correct.